### ISSUE TITLE
[FEBRUARY, '25] Update Blazor testing conference talk cross-link

### ISSUE THREAD

**Issue by guardrex** [dotnet/AspNetCore.Docs#34271]
### Description

@egil 👋 ... Hope ur well!

Question on if you feel your Oslo talk from '22 ... [How to create maintainable and testable Blazor components - Egil Hansen - NDC Oslo 2022](https://www.youtube.com/watch?v=L_n-12FglLI) ... has gone stale or not? I have it at the bottom of the Blazor *Test components* article ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/test?view=aspnetcore-9.0#additional-resources

... and I'm checking to see if it's time to pull that if it has become too old or keep it there for some time to come? I can update my tracking to check with you again in a year on it if we keep it. If you have a replacement talk that you would like me to swap in for that talk, we can do that.

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/blazor/test?view=aspnetcore-9.0

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/test.md

### Document ID

2890676f-3c63-5ab7-dd31-6b223212da30

### Article author

@guardrex

[Related Issues](https://github.com/dotnet/AspNetCore.Docs/issues?q=is%3Aissue+is%3Aopen+2890676f-3c63-5ab7-dd31-6b223212da30)

**Comment by egil** [comment]
Hi there, the session is still pretty relevant and I don't have a newer version with that level of detail on component design and component testing. 

However, next January, I will give a different session at NDC London that covered Blazor testing in general (https://sessionize.com/s/egil/blazor-testing-from-a-to-z/68495/) which is a better fit here. There is a recording of it from last year (https://youtu.be/GU_XbWjrP_g?si=1SrjeaP1T9LdFegN), but the one I give in January will have a bunch of updates, so I will leave to you to decide if you want ti replace now and then again in a few months.

**Comment by guardrex** [comment]
Yes ... I'll replace it now with the YouTube recording cross-link on an "addresses" PR, and then place this issue on hold until January for a second swap.

**Comment by guardrex** [comment]
***NOTE TO SELF***: NDC London is Jan. 27-31. https://ndclondon.com/

**Comment by egil** [comment]
Excellent. I don't know when NDC or Swetugg publishes the updated version of the session, it's usually a month or two after the event, but I will try to remember to drop by a let you know.

**Comment by guardrex** [comment]
Hello @egil ... *Happy New Year!* 🍾🎉 

I see the talk listed at ...

https://ndc-conferences-websites-eeh2hvc3deddatav.westeurope-01.azurewebsites.net/agenda/blazor-testing-from-a-to-z-06ks/0lwpqiaejj9

... but I don't see it posted for me to update the article cross-link. Is it published anywhere yet?

**Comment by egil** [comment]
NDC will usually post them a few months after the conference. Also gave the same session a week later at a Swetugg 2025 (https://youtube.com/@swetuggevents), they do tend to publish recordings faster, and the one at swetugg went slightly better, so I would want to go with that anyway 😊

**Comment by guardrex** [comment]
Great! I'll use the swetugg link when it becomes available.

**Comment by guardrex** [comment]
@egil ... I tried sending an email to them at `info@swetugg.se`, but I didn't receive a response. Do you have a contact at Swetugg to message, either here or via email, to see when they'll publish your talk? I'd like to clear these lingering 2024 issues as best I can before the storm of .NET 10 documentation updates arrive.

**Comment by egil** [comment]
@EngstromJimmy is my Swetugg person, perhaps he knows.

**Comment by guardrex** [comment]
Mmmmmm ... still nothing form Swetugg; ***however,*** the NDC London 2025 talk just went up ...

https://www.youtube.com/watch?v=p-H5fEMCB8s

I'll use that one to close this issue out. If you want it updated to Swetugg later, please open a new issue form the bottom of the article ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/test?view=aspnetcore-9.0

**Comment by guardrex** [comment]
... and of course ... if you feel the article needs work, we welcome a PR or a change set in an issue for it. An issue ***with PR*** is most helpful ⛰⛏😅, but a change set in an issue is fine to get the ball rolling. I can react to an issue and have you review the PR that I'd create to address it.
